Clients/src/presentation/components/Table/index.tsx

```
@@ -152,162 +127,176 @@ const BasicTable = ({
+    </TableHead>
+  ), [data.cols]);
+
+  const tableBody = useMemo(() => (
+    <TableBody>
+      {displayData.map((row) => (
+        <TableRow
+          sx={singleTheme.tableStyles.primary.body.row}
+          key={row.id}
+          onClick={() => {
+            console.log(`Row clicked: ${row.id}`);
+            onRowClick && onRowClick(row.id as number);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)